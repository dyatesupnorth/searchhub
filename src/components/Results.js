import React, { Component } from 'react'
import axios from 'axios';
class Results extends Component {
   
    render() {
        return (
            <div>
                <h2>Github Search Results:</h2>
            </div>
            
        )
    }
}

export default Results
