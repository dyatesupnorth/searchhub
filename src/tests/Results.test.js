import React from 'react'
import ReactDOM from 'react-dom'
import Results from '../components/Results'
import Enzyme, { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';


Enzyme.configure({ adapter: new Adapter() });

it('should render initial state', () => {
  // Shallow render
  const wrapper = shallow(<Results />);
  
  expect(toJson(wrapper)).toMatchSnapshot();
  // Expect snapshot to match
});

it('should render error state', () => {
  // Shallow render

  // Set error state

  // Expect snapshot to match
});






