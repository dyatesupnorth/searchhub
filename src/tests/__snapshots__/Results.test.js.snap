// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render initial state 1`] = `
<div>
  <h2>
    Github Search Results:
  </h2>
</div>
`;
